CHANGELOG.md

```diff
@@ -12,6 +12,20 @@ The format is based on [Keep a Changelog], and this project adheres to
 * `Fixed` for any bug fixes.
 * `Security` in case of vulnerabilities.
 
+## [0.9.2] - 2024/08/26
```

Comment:
Please put the related PRs in the changelog similar to how we did it in the past.

Reply:
Done!